File: Python/Penayangan.py
class Penayangan:
    # Constructor
    def __init__(self, judul_film="", tanggal="", jam="", studio="", harga_tiket=0):
        self.judul_film = judul_film
        self.tanggal = tanggal
        self.jam = jam
        self.studio = studio
        self.harga_tiket = harga_tiket

    # Getter & Setter
    def get_judul_film(self):
        return self.judul_film

    def set_judul_film(self, judul_film):
        self.judul_film = judul_film

    def get_tanggal(self):
        return self.tanggal

    def set_tanggal(self, tanggal):
        self.tanggal = tanggal

    def get_jam(self):
        return self.jam

    def set_jam(self, jam):
        self.jam = jam

    def get_studio(self):
        return self.studio

    def set_studio(self, studio):
        self.studio = studio

    def get_harga_tiket(self):
        return self.harga_tiket

    def set_harga_tiket(self, harga_tiket):
        self.harga_tiket = harga_tiket

    def tampilkan(self):
        print(f"Judul Film  : {self.judul_film}")
        print(f"Tanggal     : {self.tanggal}")
        print(f"Jam         : {self.jam}")
        print(f"Studio      : {self.studio}")
        print(f"Harga Tiket : {self.harga_tiket}")

File: PHP/Penayangan.php
<?php

class Penayangan {
    private $judulFilm;
    private $tanggal;
    private $jam;
    private $studio;
    private $hargaTiket;
    private $gambar; // menyimpan PATH FILE LOKAL, contoh: "uploads/poster1.jpg"

    public function __construct($judulFilm = "", $tanggal = "", $jam = "", $studio = "", $hargaTiket = 0, $gambar = "") {
        $this->judulFilm = $judulFilm;
        $this->tanggal = $tanggal;
        $this->jam = $jam;
        $this->studio = $studio;
        $this->hargaTiket = $hargaTiket;
        $this->gambar = $gambar;
    }

    // Getter & Setter
    public function getJudulFilm() {
        return $this->judulFilm;
    }

    public function setJudulFilm($judulFilm) {
        $this->judulFilm = $judulFilm;
    }

    public function getTanggal() {
        return $this->tanggal;
    }

    public function setTanggal($tanggal) {
        $this->tanggal = $tanggal;
    }

    public function getJam() {
        return $this->jam;
    }

    public function setJam($jam) {
        $this->jam = $jam;
    }

    public function getStudio() {
        return $this->studio;
    }

    public function setStudio($studio) {
        $this->studio = $studio;
    }

    public function getHargaTiket() {
        return $this->hargaTiket;
    }

    public function setHargaTiket($hargaTiket) {
        $this->hargaTiket = $hargaTiket;
    }

    public function getGambar() {
        return $this->gambar;
    }

    public function setGambar($gambar) {
        $this->gambar = $gambar;
    }
}